libethashseal/genesis/test/istanbulTransitionTest.cpp

```diff
@@ -0,0 +1,54 @@
+// Aleth: Ethereum C++ client, tools and libraries.
+// Copyright 2019 Aleth Authors.
+// Licensed under the GNU General Public License, Version 3.
+
+#include "../../GenesisInfo.h"
+
+static std::string const c_genesisInfoIstanbulTransitionTest = std::string() +
```

Comment:
Can you add Istanbul configs being a copy of Petersburg in separate PR?

Reply:
Yes